configs/module-defaults.config

```diff
@@ -55,5 +55,12 @@ params {
             publish_dir      = "bowtie2_index"
             publish_results  = "all"
         }
+        'tantan' {
+            args             = ""
+            suffix           = "tantan.txt"
```

Comment:
Not quite the way to use suffix - check out bedtools for the implementation, you basically don't want the extension, this should be fixed for the module